src/sqlfluff/core/rules/base.py

```diff
@@ -623,6 +624,10 @@ def crawl(
         return vs, raw_stack, fixes, memory
 
     # HELPER METHODS --------
+    @staticmethod
+    def _log_critical_errors(error: Exception):  # pragma: no cover
+        """This method is monkey patched into a "raise" for certain tests."""
+        pass
 
```

Comment:
Only here to all it to be patched during certain tests letting us raise during the test suite